API_Database.py

```diff
@@ -9,29 +10,31 @@ def insert(date):
     #c.execute("CREATE DATABASE API")
```

Comment:
Are we assuming that the database exists before the app starts? That's ok but do make sure you include instructions for creating the necessary database